### ISSUE TITLE
pnext: Saving db settings in Administrator gives no result

### ISSUE THREAD

**Issue by mmnosek** [Starcounter/Starcounter#3902]
I try to change database settings in Administration Panel, but changes don't create any result/error message. It doesn't matter what field I'm trying to change, what database, if the database is running or not.

Clicking "Save" button on database edit page always results in console error:

```
angular.js:9866 PUT http://192.168.0.12:8181/api/admin/databases/default/settings 403 (OK)(anonymous function) @ angular.js:9866n @ angular.js:9667f @ angular.js:9383(anonymous function) @ angular.js:13248$eval @ angular.js:14466$digest @ angular.js:14282$apply @ angular.js:14571(anonymous function) @ angular.js:21571dispatch @ jquery-2.1.0.min.js:12r.handle @ jquery-2.1.0.min.js:12
angular.js:11655 Failed to save database settings Object {data: Object, status: 403, config: Object, statusText: "OK"}(anonymous function) @ angular.js:11655(anonymous function) @ DatabaseService.js:188(anonymous function) @ angular.js:13248$eval @ angular.js:14466$digest @ angular.js:14282$apply @ angular.js:14571l @ angular.js:9698M @ angular.js:9888F.onload @ angular.js:9829
angular.js:11655 TypeError: Cannot set property 'Message' of undefined
    at DatabaseSettingsCtrl.js:97
    at DatabaseService.js:200
    at angular.js:13248
    at n.$eval (angular.js:14466)
    at n.$digest (angular.js:14282)
    at n.$apply (angular.js:14571)
    at l (angular.js:9698)
    at M (angular.js:9888)
    at XMLHttpRequest.F.onload (angular.js:9829)
```

This issue applies to two things:
- Changes in db settings should be saved
- If any error has occurred, proper message should be displayed to the user

Starcounter version: 2.3.0.4048

cc @andwah (not sure if you're a proper person :) )

**Comment by andwah** [comment]
I have not worked with pnext so im not sure what's happening there.  Just made a quick test in 'develop' branch and it works fine there.

**Comment by bigwad** [comment]
Possibly, duplicate of https://github.com/Starcounter/Starcounter/issues/3849

**Comment by chrhol** [comment]
I can confirm that this is an issue currently in v2.3 (have not tried develop though)

**Comment by dan31** [comment]
@andwah we need to fix it asap, need to release 2.3.

**Comment by andwah** [comment]
sure i will take a look first thing tomorrow.

**Comment by dan31** [comment]
@andwah thx!

**Comment by andwah** [comment]
Releated 
https://github.com/Starcounter/Starcounter/issues/3910
https://github.com/Starcounter/Starcounter/issues/3849

**Comment by andwah** [comment]
Fixed in "develop"

Where can i find step-by-step instructions on how i should put the code in 2.2 ? im using "git Extensions"
(i got good instructions from ruslan but not for this case thoug)

**Comment by mmnosek** [comment]
Looks like I have still this error in 2.3.0.5451 - should it work also there or it waits for moving to this branch?

**Comment by andwah** [comment]
the fix is only in "develop" as i mentioned i dont know how to get it to 2.3 without messing things up in the repo (i know how to cherry pick but we should not use that anylonger).

**Comment by mmnosek** [comment]
Sure, just wanted to make sure that it isn't treated as fixed in 2.3. @dan31 could you help here?

**Comment by warpech** [comment]
@andwah AFAIK cherry-pick from `develop` to `v2.3` is fine, but then you need to also merge `v2.3` -> `develop` to sync the branches.

**Comment by andwah** [comment]
The fix should now also be available in v2.3

**Comment by dan31** [comment]
Thank you @andwah!
